source/modes/GameMode.cpp

```diff
@@ -59,6 +59,7 @@
 #include <algorithm>
 #include <vector>
 #include <string>
+#include "utils/LogManager.h"
```

Comment:
OD includes should be before C/C++ standard ones. Moreover, in this case, I don't think you use LogManager